src/main/java/apprenticeship/Rules.java

```diff
@@ -0,0 +1,84 @@
+package apprenticeship;
+
+
+public class Rules {
+    private boolean hasWinner = false;
+    private boolean isADraw = false;
+    private String winner;
+    private boolean gameInProgress = true;
+
```

Comment:
Cyrus, I looked into enum, but I don't quite see how it would benefit in this context. That's why I haven't changed this. Let's come back to this later.